kotlin-native/runtime/src/main/kotlin/kotlin/native/Platform.kt

```diff
@@ -101,6 +101,12 @@ public object Platform {
     public val isFreezingEnabled: Boolean
         get() = Platform_isFreezingEnabled()
 
+    /**
+     * Representation of the name used to invoke the program executable.
+     */
+    public val programName: String
+        get() = Platform_getProgramName()
```

Comment:
What will happen if the execution is never reached `setupArgs`, but this property is accessed? E.g. when compiling Kotlin code to a native library.

A test for that would also be nice.

Reply:
Thanks for the pointer with the native library. In that case the programName doesn't even have a meaning, therefore it is now nullable. I added a test for that too using the new test infrastructure.